src/seedu/addressbook/commands/Command.java

```diff
@@ -63,6 +63,14 @@ protected ReadOnlyPerson getTargetPerson() throws IndexOutOfBoundsException {
         return relevantPersons.get(getTargetIndex() - DISPLAYED_INDEX_OFFSET);
     }
 
+    /**
+     * Determines if command is mutating the data
+     * @return true if is mutating and false if not
+     */
```

Comment:
Good to see header comments here :) Note however that the `@return …` line can be omitted for this method as it says the same thing as the first sentence.